src/rest.cpp

```diff
@@ -65,6 +65,8 @@ class RestErr
 
 extern void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry);
 extern UniValue blockToJSON(const CBlock& block, const CBlockIndex* blockindex, bool txDetails = false);
+extern UniValue mempoolInfoToJSON();
+extern UniValue mempoolToJSON(bool fVerbose = false);
```

Comment:
If I should split it into more commits, I'd prefer the other way round:

- separate `mempoolInfoToJSON` and `mempoolToJSON` and call the from RPC functions (ie. prepare the current code for REST)
- add REST functions

Should I really split it?